static/src/js/components/SearchSidebar/SearchSidebarHeader.js

```diff
@@ -1,16 +1,158 @@
 
 import './SearchSidebarHeader.scss'
 
+export const mapStateToProps = (state) => ({
+  portal: state.portal
+})
 /**
- * Renders SearchSidebarHeader.
+ * Renders SearchSidebarHeader
  */
-export const SearchSidebarHeader = () => (
-  <div className="search-sidebar-header">
-    <SearchFormContainer />
-  </div>
-)
+export const SearchSidebarHeader = ({
+  portal,
+  location,
+  onChangePath,
+  onLoadPortalConfig
+}) => {
+  const { title, logo, portalId } = portal
+
+  if (portalId === 'edsc') {
+    return (
+      <header className="search-sidebar-header">
+        <SearchFormContainer />
+      </header>
+    )
+  }
+
+  const params = parse(location.search, { parseArrays: false, ignoreQueryPrefix: true })
+  let { p = '' } = params
+  p = p.replace(/^[^!]*/, '')
+
+  const newSearch = stringify({
+    ...params,
+    p
+  })
+
+  const { primary: primaryTitle, secondary: secondaryTitle } = title
+  const { id: logoId, link: logoLink } = logo
+
+  let logoEl
+
+  if (logoId) {
+    logoEl = (
+      <div className="search-sidebar-header__thumbnail-container">
+        <div className="search-sidebar-header__thumbnail" id={logoId} />
+      </div>
+    )
+  }
+
+  if (logoLink) {
+    logoEl = (
+      <a target="_blank" rel="noreferrer" href={logoLink}>
+        {logoEl}
+      </a>
+    )
+  }
+
+  return (
+    <header className="search-sidebar-header">
+      <section className="search-sidebar-header__header">
+        {logoEl}
+        <div className="search-sidebar-header__primary">
+          <h2
+            className="search-sidebar-header__heading"
+            title={`${primaryTitle}${secondaryTitle && ` (${secondaryTitle})`}`}
+          >
+            {primaryTitle}
+            {
+              secondaryTitle && (
+                <>
+                  {' '}
+                  <span className="search-sidebar-header__heading-secondary">
+                    {`(${secondaryTitle})`}
+                  </span>
+                </>
+              )
+            }
+          </h2>
+          {/* <PortalLinkContainer
+            type="button"
+            className="search-sidebar-header__button"
+            // bootstrapVariant="light"
+            icon={FaTimes}
+            label="Exit Portal"
+            dataTestId="exit-portal-button"
+        // TODO: passes search params
+            to={{
+              pathname: '/',
+              search: newSearch
+            }}
+            // onClick={() => { onChangePath(`/search${newSearch}`) }}
+          >
+            Exit Portal
+          </PortalLinkContainer> */}
+
+          <Link to={{
+            pathname: '/',
+            search: newSearch
+          }}
+          >
+            <Button
+              className="search-sidebar-header__button"
+              icon={FaTimes}
+              variant="link"
+              title="Exit Portal"
+              label="Exit Portal"
+              onClick={() => {
+                onLoadPortalConfig()
```

Comment:
`onLoadPortalConfig` should only be called from the PortalContainer, it isn't needed to get this link to work